sample-tests/src/test/java/com/github/wasiqb/boyka/api/requests/BookingDataBuilder.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * MIT License
+ *
+ * Copyright (c) 2022 Wasiq Bhamla
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ */
+
+package com.github.wasiqb.boyka.api.requests;
+
+import java.text.SimpleDateFormat;
+import java.util.concurrent.TimeUnit;
+
+import com.github.javafaker.Faker;
+
+/**
+ * @author Faisal Khatri
+ * @since 22/07/2022
+ */
+public class BookingDataBuilder {
+    public BookingData bookingDataBuilder () {
```

Comment:
Add JavaDoc to all public class and methods.

Reply:
updated